fedimintd/src/bin/main.rs

```diff
@@ -53,6 +53,8 @@ async fn main() {
         }
     }
 
+    info!("Starting fedimintd (version: {})", CODE_VERSION);
```

Comment:
Clippy will complain new new version of Rust that you can put `CODE_VERSION` into `{}` directly, so you might as well fix it already. :)

Reply:
@dpc Can we add this new clippy version to CI run on PRs? It seems to run for MacOS, so it would fail on master after merge.

Reply:
Done